docs/root/start/install.rst

```diff
@@ -128,6 +128,18 @@ You can install Envoy on Mac OSX using the official brew repositories, or from
 	 `Get Envoy <https://www.getenvoy.io/>`__ by adding the ``--HEAD`` flag to
 	 the install command.
 
+.. _start_install_windows:
+
+Install Envoy on Windows
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can run Envoy using the official Windows development Docker image.
```

Comment:
Do we want to mention downloading the binaries from the CI? Or is it going to be too hacky?

Reply:
i think these are user docs - also i think the artefacts in ci are ephemeral

Reply:
I'm also unsure if the instructions would always work if say the windows.release stage fails, which I think would cause unnecessary confusion. I'm leaning towards not including such instructions. It might be better to leave that for another PR anyway...

Reply:
I'll mark this comment as resolved for now.